food/api/app/serializers/place_serializer.rb

```diff
@@ -10,12 +10,9 @@ class PlaceSerializer < ActiveModel::Serializer
     location
     visit_radius
     reservations_url
+    category_names
   ])
 
-  attribute :cached_nabes, key: :nabes
-
-  attribute :cached_categories, key: :categories
```

Comment:
@achainan As of the latest changeset (#228) the app needs these fields to cache categories/nabes locally (if only for their UUIDS); if we decide to pare down the `places` payload like this we'll need to marry the change w/ supplemental requests for the same data.

Reply:
Ok reverted this portion. Please review and merge.